packages/app/src/pages/SwapPage/jotai.ts

```diff
@@ -9,3 +10,20 @@ export const swapActiveInputAtom = atom<ActiveInput>(ActiveInput.from);
 export const swapCoinsAtom = atom<[Coin | null, Coin | null]>([assets[0], null]);
 export const swapAmountAtom = atom<bigint | null>(null);
 export const swapIsTypingAtom = atom<boolean>(false);
+
+export const useSetTyping = () => {
```

Comment:
we could have this not just for `set` but for is typing in general, so we could use:

```js
const [isTyping, setTyping] = useIsTyping()
```

Because you have a hook here, but in another place, we still using `useAtomValue(isTypingAtom)`.
So, since you created a hook for this, maybe you can implement it for both cases.